.esbuild/util.cjs

```diff
@@ -39,17 +44,35 @@ exports.esmBuild = (override = { minify: true }) => {
 };
 
 /**
- * @param {Options & { core?: boolean }} override
- * @returns {Options}
+ * Build options for mermaid.core.* build.
+ *
+ * This build does not bundle `./node_modules/`, as it is designed to be used
+ * with Webpack/ESBuild to merge webpack into a website.
```

Comment:
```suggestion
 * with Webpack/ESBuild/Vite to use mermaid inside an app/website.
```

Good catch! I'll change to the following, since now that I've tested this with `mermaid-live-editor`, I know that it works with Vite.